substrate/client/rpc-spec-v2/src/transaction/api.rs

```diff
@@ -28,10 +28,33 @@ pub trait TransactionApi<Hash: Clone> {
 	///
 	/// See [`TransactionEvent`](crate::transaction::event::TransactionEvent) for details on
 	/// transaction life cycle.
+	///
+	/// # Unstable
+	///
+	/// This method is unstable and subject to change in the future.
 	#[subscription(
 		name = "transactionWatch_unstable_submitAndWatch" => "transactionWatch_unstable_watchEvent",
 		unsubscribe = "transactionWatch_unstable_unwatch",
 		item = TransactionEvent<Hash>,
 	)]
 	fn submit_and_watch(&self, bytes: Bytes);
 }
+
+#[rpc(client, server)]
+pub trait TransactionBroadcastApi<Hash: Clone> {
+	/// Broadcast an extrinsic to the chain.
+	///
+	/// # Unstable
+	///
+	/// This method is unstable and subject to change in the future.
+	#[method(name = "transaction_unstable_broadcast", blocking)]
```

Comment:
I don't understand why this is "blocking", AFAIU this should just be an ordinary method i.e, doesn't wait sync I/O such as the storage DB.

Reply:
Good point; I had just thought "oh, it's not async so this is ok" but I reminded myself that blocking means it runs the task in `tokio::task::spawn_blocking`, which doesn't appear to be necessary here since not much work happens outside the future

Reply:
Yep that makes sense, I think I just copied the method attribute from somewhere else. Have made it non-blocking